bin/improver-orographic-enhancement

```diff
@@ -0,0 +1,137 @@
+from improver.orographic_enhancement import OrographicEnhancement
+from improver.utilities.load import load_cube
+from improver.utilities.save import save_netcdf
+from improver.utilities.filename import generate_file_name
+
+
+def main():
+    """Calculate orographic enhancement of precipitation from model pressure,
+    temperature, relative humidity and wind input files"""
+
+    parser = ArgParser(description='Calculate orographic enhancement using the'
+                       ' ResolveWindComponents() and OrographicEnhancement() '
+                       'plugins. Outputs data on the high resolution orography'
+                       ' grid and regridded to the coarser resolution of the '
+                       'input diagnostic variables.')
+
+    parser.add_argument('temperature_filepath', metavar='TEMPERATURE_FILEPATH',
+                        help='Full path to input NetCDF file of temperature on'
+                        ' height levels')
+    parser.add_argument('humidity_filepath', metavar='HUMIDITY_FILEPATH',
+                        help='Full path to input NetCDF file of relative '
+                        'humidity on height levels')
+    parser.add_argument('pressure_filepath', metavar='PRESSURE_FILEPATH',
+                        help='Full path to input NetCDF file of pressure on '
+                        'height levels')
+    parser.add_argument('windspeed_filepath', metavar='WINDSPEED_FILEPATH',
+                        help='Full path to input NetCDF file of wind speed on '
+                        'height levels')
+    parser.add_argument('winddir_filepath', metavar='WINDDIR_FILEPATH',
+                        help='Full path to input NetCDF file of wind direction'
+                        ' on height levels')
+    parser.add_argument('orography_filepath', metavar='OROGRAPHY_FILEPATH',
+                        help='Full path to input NetCDF high resolution '
+                        'orography ancillary. This should be on the same or a '
+                        'finer resolution grid than the input variables, and '
+                        'defines the grid on which the orographic enhancement '
+                        'will be calculated.')
+    parser.add_argument('output_dir', metavar='OUTPUT_DIR', help='Directory '
+                        'to write output orographic enhancement files')
+    parser.add_argument('--boundary_height_metres', type=float, default=1000.,
+                        help='Model height level to extract variables for '
+                        'calculating orographic enhancement, as proxy for '
+                        'the boundary layer.')
+
+    args = parser.parse_args()
+
+    # read input variables
+    bh = args.boundary_height_metres
+    constraint = iris.Constraint(height=lambda h: bh - 0.1 < h < bh + 0.1)
+
+    load_error_msg = 'No data available from {} at height {} m'
+
+    temperature = load_cube(args.temperature_filepath, constraint)
```

Comment:
Won't this fail anyway, with an error like this:
```ValueError: No cubes found using contraints...```
In which case we won't reach the nice error message. Could do with a try/except, or could just live with the built in message.